client/FxAccountClient.js

```diff
@@ -871,6 +871,7 @@ define([
    * @param {String} email
    * @param {String} newPassword
    * @param {Object} oldCreds This object should consists of `oldUnwrapBKey`, `keyFetchToken` and `passwordChangeToken`.
+   * @param {Boolean} [options.keys] If `true`, calls the API with `?keys=true` to get the keyFetchToken
```

Comment:
Resubmitting the comment for visibility: 

Thanks for the pull request!

This line needs to be changed from:
```
* @param {Boolean} [options.keys] If `true`, calls the API with `?keys=true` to get the keyFetchToken 
```
to

```
* @param {Object} keys This object should contain the unbundled keys
```